modules/apps/roles/roles-admin-test/src/testIntegration/java/com/liferay/roles/admin/internal/search/test/RoleIndexerTest.java

```diff
@@ -66,7 +66,7 @@ public void testSearchByName() throws Exception {
 
 	@Test
 	public void testSearchByType() throws Exception {
-		Role role = RoleTestUtil.addRole(RoleConstants.TYPE_DEPOT);
+		Role role = RoleTestUtil.addRole(RoleConstants.TYPE_ACCOUNT);
```

Comment:
as you can see also in this test (that with your change I think it fails) on line L144 there is a dedicated assert for account roles that are not in the list unless you specify the type in the search